Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -23,23 +23,7 @@ def linkage
         result.display_test_output
         Homebrew.failed = true if result.broken_dylibs?
         if OS.linux?
```

Comment:
Usage of `OS.linux?` was already in this file, so I'll go with that given what Shaun said above.